cmd/helper.go

```diff
@@ -31,3 +35,74 @@ func homeDir() (string, error) {
 
 	return cuser.HomeDir, nil
 }
+
+func Abs(n int64) int64 {
+	if n < 0 {
+		return -n
+	}
+	return n
+}
+
+type LightOperation struct {
+	PagingToken     string `json:"paging_token"`
+	CreatedAt       string `json:"created_at"`
+	CreatedAtParsed time.Time
+}
+
+func parseLightOperation(operation interface{}) (op LightOperation, err error){
+	b, err := json.Marshal(operation)
+	err = json.Unmarshal(b, &op)
+	if err != nil {
+		return op, fmt.Errorf("Errror while Unmarshaling operation")
```

Comment:
errors should have user friendly messages, i.e. in this case "could not parse light operation (...explanation what a light operation is)"